defaults/main.yml

```diff
@@ -108,9 +120,11 @@ postgresql_external_pid_file: "{{ postgresql_pid_directory }}/{{ postgresql_vers
 #------------------------------------------------------------------------------
 # CONNECTIONS AND AUTHENTICATION
 #------------------------------------------------------------------------------
-
+pgdb_iface: "{{ lookup('env','PGDB_IFACE') | default(ansible_default_ipv4.interface, true) }}"
```

Comment:
Do we want to be relying on environment variables?  Shouldn't we get this from an inventory?